webapp/channels/src/components/common/carousel/carousel.tsx

```diff
@@ -4,7 +4,7 @@
 import React, {useEffect, useState} from 'react';
 
 import CarouselButton, {BtnStyle} from './carousel_button';
-import './carousel.scss';
```

Comment:
Just curious why we're renaming to index.scss instead of carousel.scss. It's not super obvious as a filename